src/MarDyn.cpp

```diff
@@ -146,6 +149,11 @@ int main(int argc, char** argv) {
 	vector<string> args = op.args();
 
 	global_log->info() << "Running ls1-MarDyn version " << MARDYN_VERSION << endl;
+
+#ifdef MARDYN_AUTOPAS
+	global_log->info() << "Built with AutoPas version " << AutoPas_VERSION << std::endl;
+#endif
```

Comment:
We already print the AutoPas Version in the AutoPas log at info level. This is invoked through AutoPas::init(), so not reasonably circumventable. Are you sure you want it here too?

Reply:
yep :)
didn't find it, so better to have it here, too.